dashboard/lib/app.js

```diff
@@ -82,6 +138,32 @@ function compile(str, path) {
 }
 
 
+function loadRegionsFromSettings() {
```

Comment:
maybe I should add this function to global.regionsCache, something like update or getRegions